src/Microsoft.OData.Client/DataServiceQueryOfT.cs

```diff
@@ -314,14 +314,27 @@ public DataServiceQuery<TElement> Expand<TTarget>(Expression<Func<TElement, TTar
         /// <returns>A new <see cref="T:Microsoft.OData.Client.DataServiceQuery`1" /> object that has the inline count option set.</returns>
         public DataServiceQuery<TElement> IncludeTotalCount()

```

Comment:
>IncludeTotalCount [](start = 42, length = 17)

TotalCount is OData V3 term, OData V4 is using Count. Consider to refactor as "IncludeCount()"?

Reply:
@xuzhg Do we still keep the IncludeTotalCount() for backwards compatibility? cc @mikepizzo

Reply:
Assuming that, for OData V4, setting IncludeTotalCount() sets $count=true, then we should keep it for backward compatibility.

Reply:
Or maybe we can mark the old one as [obsoleted] and add new one as "IncludeCount()". 

I agree both. 

---
In reply to: [381424145](https://github.com/OData/odata.net/pull/1658#discussion_r381424145) [](ancestors = 381424145)

Reply:
@mikepizzo @xuzhg 
I have added `QueryOperationResponse<T>.Count` and `IncludeCount`
I have deprecated `QueryOperationResponse<T>.TotalCount` and `IncludeTotalCount`